### DISCUSSION TITLE
AHC of MoS2 monolayer

### DISCUSSION THREAD

**Discussion by sourentpty** [wannier-berri/wannier-berri#313] [Q&A]
Dear Prof. Stepan-tsirkin,
I, want to calculate the AHC and berry curvature of monolayer MoS2. I have generated the basic requirements from the wannier90 run (i.e., amm, mnm, win, eig, and _tb.dat etc. ). Now I want calculate berry curvature and AHC using Wannier-berri. Can you help me or share any example related to the 2D materials. 

Thank you in advance!
Best!

**Comment by stepan-tsirkin** [comment]
There is not much difference for 2D materials, compared to 3D, just follow the usual tutorials:
https://github.com/wannier-berri/WannierBerri-tutorial/tree/main/tutorials/1_basic
https://tutorial.wannier-berri.org/tutorials/1_basic/tutorial-wb-basic-solution.html

It is recommended to set periodic = [True, True, False] for 2D system, but apart from that - all the same
https://docs.wannier-berri.org/en/master/docs/system.html#wannierberri.system.System

**Comment by gokaran-tech** [comment]
Dear Prof Stepan
What about the scaling factor in 2d? It seems WBerri is still using 3d Mesh with k=1 in the z direction. So the result should be multiplied by 2pi/c   where c is the length in the Z direction in   Ang?

**Reply by stepan-tsirkin** [reply]
Yes, you are right.
